front/pages/w/[wId]/builder/data-sources/[name]/index.tsx

```diff
@@ -725,6 +727,7 @@ interface ConnectorUiConfig {
   addDataButtonLabel: string | null;
   displaySettingsButton: boolean;
   guideLink: string | null;
+  onPostPermissionsUpdate?: () => ReactNode;
```

Comment:
here, you're passing content
but convention is usually `onSomething` = handler on event something, `somethingCallback` => happens after something is done
I would do `postPermissionsUpdateMessage?: ReactNode` here
More generic but needs a bit refactoring thinking would be `permissionsUpdateCallback?: () => void` (but not sure it can work

Reply:
Agreed, changing.

Reply:
Changed to just return a string as well.